.rubocop_todo.yml

```diff
@@ -585,12 +581,24 @@ Style/FormatStringToken:
   Exclude:
     - 'vendor/engines/bulk_email/spec/services/bulk_email/content_generator_spec.rb'
 
-# Offense count: 1343
+# Offense count: 12
 # Cop supports --auto-correct.
 # Configuration parameters: EnforcedStyle.
 # SupportedStyles: when_needed, always, never
 Style/FrozenStringLiteralComment:
-  Enabled: false
+  Exclude:
+    - 'public/files/000/000/001/original/alpha_survey.rb'
```

Comment:
I think these are some local files that I had uploaded as order forms or something. These probably shouldn't be here.